src/documentation/0053-node-difference-dev-prod/index.md

```diff
@@ -1,14 +1,17 @@
 ---
 title: Node.js, the difference between development and production
 description: 'Learn how to set up different configurations for production and development environments'
-authors: flaviocopes, MylesBorins, fhemberger, LaRuaNa, ahmadawais
+authors: flaviocopes, MylesBorins, fhemberger, LaRuaNa, ahmadawais, 19shubham11
 section: Getting Started
 ---
 
-You can have different configurations for production and development environments.
+While working on a real project you may want your server to interact with several `outside` systems, that may be databases, messaging queues or even external APIs. In this scenario, a good convention to follow is to always have multiple environment variables and configurations associated with them.
 
-Node.js assumes it's always running in a development environment.
-You can signal Node.js that you are running in production by setting the `NODE_ENV=production` environment variable.
+You may have `development`, `QA`, `staging` and `production` environments depending upon your scale and use case, each of these environments should ideally have different configurations (database URLs and passwords, API access keys, etc.), for example, you may not want to pollute your production database while fixing a bug locally! This is where different environments come into play and the most commonly recommended way of doing this in Node.js is setting up an environment variable `NODE_ENV` and trigger different configs based on that.
+
+More on [environment variables](https://en.wikipedia.org/wiki/Environment_variable)
+
+You can signal Node.js that you are running in a particular environment by setting the `NODE_ENV=$environment` environment variable.
```

Comment:
I think it would be better if this made it explicit that you are signaling code that is being run by Node, and not signaling Node itself - Node doesn't care what this variable is set to.

I don't know whether this is just me being too literal for the level of tutorial this is.

Reply:
How about this?

```suggestion
You can specify that you are running in a particular environment by setting the `NODE_ENV=$environment` environment variable.
```

Reply:
Looks to be alright @19shubham11. :tada:

I'd leave the `in` there. It's more accurate.

```suggestion
You can specify that you are running in a particular environment by setting the `NODE_ENV=$environment` environment variable.
```